test/fixtures/linter/rules/NoDeprecatedApi/testsuite.qunit.ts

```diff
@@ -0,0 +1,38 @@
+import { ui5 } from "sap/viz/library";
```

Comment:
is this line here intentionally? If so, please describe why, as it might not be totally obvious to everyone.

Reply:
Removed.